examples/seismic/abc_methods/04_habc.ipynb

```diff
@@ -0,0 +1,1617 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# 4.1 - Introduction\n",
```

Comment:
Proper title before intro.

Reply:
Just a mistake! I already corrected this.